### PR TITLE
feat(core): logging message for sync status

### PR DESCRIPTION
**Pull request by stephhuynh18**
Temp logging message before we have a node status endpoint

### CODE REVIEW

**Review by PaulLeCam — APPROVED** [review]

**Review by oed — APPROVED** [review]
lgtm

**Comment by stbrody** [comment]
FYI @nathanielc as this might be relevant to getting the data on running syncs for use by the nodeStatus endpoint

File: packages/core/src/sync/sync-api.ts [hunk 1]
```diff
@@ -234,6 +266,74 @@ export class SyncApi implements ISyncApi {
...
+    const pendingSyncJobs = pendingJobs[HISTORY_SYNC_JOB] || []
+
+    const syncStatus: SyncStatus = {
+      activeSyncs: historySyncJobs.map((job) => {
+        const jobData = job.data as SyncJobData
+        return {
+          currentBlock: jobData.fromBlock,
+          endBlock: jobData.toBlock,
+          models: jobData.models,
+          created: job.createdOn,
```
[the remark this answers is not in the document]
Comment (stephhuynh18) [reply packages/core/src/sync/sync-api.ts:288]: createdOn is the when the sync was requested, startOn is the when the sync was started

File: packages/core/src/sync/sync-api.ts [hunk 2]
```diff
@@ -234,6 +266,74 @@ export class SyncApi implements ISyncApi {
...
+          startTime: job.startedOn,
+        }
+      }),
+
+      continuousSync:
+        continuousSyncJobs.length > 0
+          ? continuousSyncJobs.map((job) => {
+              const jobData = job.data as SyncJobData
+              return {
+                latestBlock: this.currentBlock,
```
[the remark this answers is not in the document]
Comment (stephhuynh18) [reply packages/core/src/sync/sync-api.ts:298]: I don't think the continuous sync job needs a start block as each job is for one block (the block that was heard - confirmations)

**Pull request merged**